core/common/src/main/java/alluxio/conf/PropertyKey.java

```diff
@@ -2444,6 +2444,15 @@ public String toString() {
           .setConsistencyCheckLevel(ConsistencyCheckLevel.WARN)
           .setScope(Scope.WORKER)
           .build();
+  public static final PropertyKey WORKER_BLOCK_LOCK_TIMEOUT =
+      new Builder(Name.WORKER_BLOCK_LOCK_TIMEOUT)
+          .setDefaultValue("3min")
```

Comment:
Does this mean all block reads or writes must complete within this time? In high load situations, I have seen remote reads of a block take almost 30 minutes to finish reading. In this scenario, what would happen?

Reply:
I add a lockHeartbeat (basically updateLockAccessTime) in the BlockReadHandler. Every read from remote with update the lock access time. But yeah, for the `ShortCurcuitReadHandler`, I don't have ways to track the locks, so it may be better to set this value to be bigger. If `ShortCircuitBlockReadHandler` takes too long, they may be error out as blocks are removed..